cgroups/systemd/apply_systemd.go

```diff
@@ -196,6 +209,22 @@ func writeFile(dir, file, data string) error {
 	return ioutil.WriteFile(filepath.Join(dir, file), []byte(data), 0700)
 }
 
+func joinCpuQuota(c *cgroups.Cgroup, pid int) error {
```

Comment:
I think it makes sense to have a single function for a single cgroup sub-system.